### PR TITLE
feat: wiring of rooms sdk to markup

### PR DESCRIPTION
**Pull request by alexandra-marin**
#### Relevant info and issue/PR links 
 https://app.clubhouse.io/peerio/story/7833/medcryptor-desktop-wiring-of-rooms-sdk-to-markup
#### Testing instructions  

----
### Repository owner

- [ ] Was tested and can be merged.
- [ ] PR name follows conventional changelog format.

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### CODE REVIEW

**Comment by flohdot** [comment]
merged, and made a few small changes after testing what patient rooms look like from MC mobile and Peerio

File: src/models/chats/chat-head.js [hunk 1]
```diff
@@ -45,6 +45,14 @@ class ChatHead extends SyncedKeg {
      */
     @observable spaceName = null;
 
+    /**
+     * @member {string} nameInSpace
+     * @memberof ChatHead
+     * @instance
+     * @public
+     */
+    @observable nameInSpace = null;
```
Comment (anri-asaturov) [inline src/models/chats/chat-head.js]: because room will be named differently for patient and docs? @flohdot ?

Comment (flohdot) [reply src/models/chats/chat-head.js]: yes, otherwise it looks really messed up for everyone else

Comment (flohdot) [reply src/models/chats/chat-head.js]: and I'd rather leave the default `name` for use with the default peerio client than the other way around

Comment (anri-asaturov) [reply src/models/chats/chat-head.js]: how would rename work then? 2 inputs?

Comment (flohdot) [reply src/models/chats/chat-head.js]: my thought is rename is just for internal purposes (`nameInSpace`). gonna create that task.

File: src/models/chats/chat-store.js [hunk 2]
```diff
@@ -249,6 +251,16 @@ class ChatStore {
...
 
+    /**
+     * currently selected/focused space.
+     * @member {string} activeSpace
+     * @type {string} activeSpace
+     * @memberof ChatStore
+     * @instance
+     * @public
+     */
+    @observable activeSpace = null;
```
Comment (anri-asaturov) [inline src/models/chats/chat-store.js]: I tried to direct you guys towards _not_ creating a complicated state of activeSpace + activeChat, but failed. 
Active space is really easy to detect -  `chatStore.activeChat.isInSpace` and no need to manage another codependent flag. Unless you can have a state where no chat is selected? zerostate for spaces exists? @flohdot

Comment (flohdot) [reply src/models/chats/chat-store.js]: there is a zero state for spaces

Comment (anri-asaturov) [reply src/models/chats/chat-store.js]: but isn't space always created with N chats in it automatically? why does it need a zero state if it's never empty

Comment (flohdot) [reply src/models/chats/chat-store.js]: it was designed with a zero state, check out the mocks...

Comment (flohdot) [reply src/models/chats/chat-store.js]: it's mostly for orientation/explanation

Comment (anri-asaturov) [reply src/models/chats/chat-store.js]: These changes go into Peerio logic deeper than we would want ( routing on desktop..

**Pull request merged**